### DISCUSSION TITLE
Retrieving Data Tutorial

### DISCUSSION THREAD

**Discussion by schummd** [filecoin-project/community#574] [Developers]
I have been following [this tutorial](https://docs.filecoin.io/get-started/store-and-retrieve/retrieve-data/) to test retrieval of data from Filecoin. 

However, when attempting to get deal information for `f16wc2ees6nkmk4pguoaaly4zdkeemaajjbv7aqgq` there is nothing available (https://filecoin.tools/f16wc2ees6nkmk4pguoaaly4zdkeemaajjbv7aqgq). 

Is there an issue or such deal does not actually exist? Probably the tutorial page should be updated.

**Comment by TippyFlitsUK** [comment]
Hi @schummd  👋 
That is the old docs site which is being phased out. Please head over to https://lotus.filecoin.io/tutorials/lotus/store-and-retrieve/retrieve-data/
